samples/msgext-search-sso-config/csharp/AppManifest/manifest.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
```

Comment:
Why are we adding new manifest for this sample?

Reply:
We have removed the new manifest, as it is not required.